src/Controls/src/Core/Handlers/Items/Android/MauiCarouselRecyclerView.cs

```diff
@@ -594,14 +591,23 @@ protected override void OnMeasure(int widthMeasureSpec, int heightMeasureSpec)
 
 			base.OnMeasure(widthMeasureSpec, heightMeasureSpec);
 		}
-	}
 
-	class CarouselViewwOnGlobalLayoutListener : Java.Lang.Object, ViewTreeObserver.IOnGlobalLayoutListener
-	{
-		public EventHandler<EventArgs> LayoutReady;
-		public void OnGlobalLayout()
+		class CarouselViewOnGlobalLayoutListener : Java.Lang.Object, ViewTreeObserver.IOnGlobalLayoutListener
```

Comment:
I made this a nested private class, so it could access the `LayoutReady` private method.